avro-fastserde/src/jmh/java/com/linkedin/avro/fastserde/FastAvroSerdesBenchmark.java

```diff
@@ -6,6 +6,7 @@
 import com.linkedin.avroutil1.compatibility.AvroCompatibilityHelper;
 import java.io.ByteArrayInputStream;
 import java.util.HashMap;
+import java.util.List;
```

Comment:
Not needed?

Reply:
yes, removed it.

Reply:
Yes, we plan to port these to other primitive arrays.